pkgs/applications/virtualization/distrobox/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "distrobox";
+  version = "1.2.13";
+
+  src = fetchurl {
+    url = "https://github.com/89luca89/distrobox/archive/refs/tags/1.2.13.tar.gz";
+    sha256 = "00x6rrw1z2z2w1aw2l2v773n25rd25d8fmvz45i188gjp316kyq1";
+  };
```

Comment:
Maybe we can use `fetchFromGitHub`?

Reply:
Hello, thx for the review! hm, reviewers often ask that, I dunno why exactly, as I often prefer stable releases than just grabbing stuff from master branch.
Can u elaborate on your rationale?

Reply:
`fetchFromGitHub` grabs the release tarball. E.g., 

```
  src = fetchFromGitHub {
    owner = "89luca89";
    repo = pname;
    rev = version;
    hash = "sha256-ZP5qtbjDtBZ2uT1DOSvSnfHyTlirr0oCEXhiLO1ydz0=";
  };
````

Reply:
oaah this seems nice, I will try!